Bookish/Views/Catalog/BookEntry.cshtml

```diff
@@ -2,7 +2,7 @@
 @model Bookish.Views.Catalog.BookEntry
 
 @{
-    Layout = null;
+    ViewBag.Title = "Book Entered";
```

Comment:
Any reason you're not using `ViewData['Title']`?

Reply:
It breaks it and I have no idea why! When I try and enter a new book with viewData["Title"] I get this error: 
![image](https://user-images.githubusercontent.com/43523985/178749003-6534532b-426e-4af2-a122-fe8bd915074d.png)
but with the viewBag it seems to work.

Reply:
Interesting, not seen this before! Something to look into...